code/game/objects/effects/decals/decal.dm

```diff
@@ -23,8 +22,8 @@
 	if(B && B.loc == loc)
 		qdel(src)
 
-/obj/effect/decal/proc/NeverShouldHaveComeHere(turf/T)
-	return isclosedturf(T) || (isgroundlessturf(T) && !SSmapping.get_turf_below(T))
+/obj/effect/decal/proc/NeverShouldHaveComeHere(turf/here_turf)
```

Comment:
Please add a comment what this proc does because meme name doesn't help when it's functional